scripts/package.ps1

```diff
@@ -21,8 +22,7 @@ Function Ensure-Nuget-Exists {
 Write-Host "** Building all NuGet packages. **"
 foreach ($file in [System.IO.Directory]::EnumerateFiles("$repoRoot\src", "project.json", "AllDirectories")) {
     Write-Host "Creating NuGet package for $file..."
-    $dateSuffix = [System.DateTime]::Now.ToString('eyyMMdd-1')
```

Comment:
I think we need a fallback if `$BuildVersion = $null` so that we can build locally as well.